File: text/dart/read/text_read.dart
/*
	text_read.dart

					Jan/09/2021
*/
// -------------------------------------------------------------
import 'dart:async';
import 'dart:io'; 

import	'/var/www/data_base/common/dart_common/text_manipulate.dart';
// -------------------------------------------------------------
main (List<String> arguments) {
	stderr.writeln ("*** 開始 ***");

	String file_name = '${arguments[0]}';

	File file = new File (file_name);
	Future future = file.readAsString ();

	future.then((content)
		{
		var dict_aa = str_to_dict_proc (content,'\t');

		dict_display_proc (dict_aa);

		stderr.writeln ("*** 終了 ***");
		}); 
}

// -------------------------------------------------------------

File: text/nashorn/read/text_read.js
#! /usr/bin/env jjs
// ------------------------------------------------------------------
//
//	text_read.js
//
//						Jan/04/2017
//
// ------------------------------------------------------------------
load ("/var/www/data_base/common/nashorn_common/text_manipulate.js")
load ("/var/www/data_base/common/nashorn_common/file_io.js")
// ------------------------------------------------------------------

print	("*** 開始 ***")
print("Arguments: " + arguments)

file_in=arguments[0]

var dict_aa = text_read_proc (file_in)
dict_display_proc (dict_aa)

print	("*** 終了 ***")
// ------------------------------------------------------------------
